docs/working-with-groups.md

```diff
@@ -98,6 +100,12 @@ for (Group actGroup : searchResult.Resources()) {
 //...
 ```
 
+It is possible to filter the attributes of the returned groups by supplying the attributes as parameters to the method:
+
+```java
+SCIMSearchResult<Group> searchResult = oConnector.getAllGroups(accessToken, "displayName", "meta.created");
```

Comment:
`OsiamConnector#getAllGroups` returns `List<Group>` not `SCIMSearchResult<Group>`.